apps/shuffle/butterfly_network.py

```diff
@@ -22,50 +22,51 @@
         await asyncio.sleep(1)
 
 
-async def batchBeaver(ctx, Xs, Ys, As, Bs, ABs):
-    Ds = await (Xs - As).open()  # noqa: W606
-    Es = await (Ys - Bs).open()  # noqa: W606
+async def batch_beaver(ctx, xs, ys, as_, bs_, abs_):
+    ds = await (xs - as_).open()  # noqa: W606
+    es = await (ys - bs_).open()  # noqa: W606
 
```

Comment:
I think it's useful to have a way to distinguish between the "public" values of reconstructed secrets and the "shared" values of secrets themselves. It's used a bit inconsistently here, but the idea has been `D` is a public value while `a,b,x,y,ab` are all secrets. Is there any other convention we could use along these lines that's pep8 compatible? Having the underscore doesn't seem to connote much

Reply:
The reason you see the underscore is because `as` is a reserved keyword in python and as per pep8 a trailing underscore can be used to avoid the name collision.

I understand your point wrt the naming convention on paper, do you think it would be better to add a `#noqa` to ignore this and change it back to capitals?